sql/original_update_tests.md

```diff
@@ -0,0 +1,48 @@
+# Original Update Tests
+This file is a port of the original hardcoded update-tester tests to markdown, similar to how sql-doctester works.
+
+These markdown files are parsed in the same way as sql-doctester would with the addition of two fields: `creation` and `validation`. (Names are not final).
+
+Tests marked `creation` will be ran during the object creation step, tests marked validation will be ran during the validation step.
+ 
+
+ 
+## Object Creation Step
+ 
+Currently, `SET TIME ZONE 'UTC';` will cause tests to fail if used in the `validation` step, so it should used in `creation` if needed.
+```sql,creation
+SET TIME ZONE 'UTC';
```

Comment:
Won't we always want this?  I think update-tester should take care of this across the board.

Reply:
I am not sure if we always want it but if we do I can definitely have update-tester handle it

Reply:
If we don't, tests will vary as we all have different local timezone settings :)

If we had some test that needed to test something in some other specific timezone, it should be able to override, as i think the last SET wins.  And would that leak?  Maybe update-tester needs to do this before running every script?

Reply:
Changing code to set timezone to UTC before running any scripts.